support-frontend/assets/pages/subscriptions-landing/copy/subscriptionCopy.jsx

```diff
@@ -82,18 +68,18 @@ export type ProductCopy = {
 
 const abTest = null;
 
-const getPrice = (countryGroupId: CountryGroupId, product: SubscriptionProduct) => {
-
-  if (flashSaleIsActive(product, countryGroupId)) {
-    return getDisplayFlashSalePrice(product, countryGroupId, Monthly);
-  }
-
-  if (subscriptionPricesForDefaultBillingPeriod[product][countryGroupId]) {
-    return `${displayPrice(product, countryGroupId)}`;
-  }
-
-  return '';
-};
+// const getPrice = (countryGroupId: CountryGroupId, product: SubscriptionProduct) => {
+//
+//   if (flashSaleIsActive(product, countryGroupId)) {
+//     return getDisplayFlashSalePrice(product, countryGroupId, Monthly);
+//   }
+//
+//   if (subscriptionPricesForDefaultBillingPeriod[product][countryGroupId]) {
+//     return `${displayPrice(product, countryGroupId)}`;
+//   }
+//
+//   return '';
+// };
```

Comment:
Is this stuff commented out because it will need to be reinstated once the sale is over?

Reply:
Yes